### ISSUE TITLE
Customization matters

### ISSUE THREAD

**Issue by uteshevia** [uteshevia/toolkitex#6]
It's the main project breakthrough.
All the primary parameters are to be altered easily being provided with different general presets,
such as one/twoside option, different page margins, headers and numerations etc.

**Comment by gkorepanov** [comment]
Do you want to rewrite TeX? We are always facing the same problem of inventing a bicycle when getting up to the point of customization.

**Comment by uteshevia** [comment]
No, just to provide the interface to perform such ordinary options like *set document font size* or *show total page count*.

**Comment by gkorepanov** [comment]
And why do we need it? It might be implemented with corresponding LaTeX commands. At which point you are going to stop?

**Comment by uteshevia** [comment]
At this point, literally:
```\documentclass[10pt, twoside, margins=narrow, totalpages, enumresume]{urticle}
```

**Comment by uteshevia** [comment]
At this point, literally:
```
\documentclass[10pt, twoside, margins=narrow, totalpages, enumresume]{urticle}
```

**Comment by gkorepanov** [comment]
Well, why don't define geometry as well?

**Comment by uteshevia** [comment]
This can be duscussed.
